IBAnimatable/FadeAnimator.swift

```diff
@@ -64,9 +66,9 @@ extension FadeAnimator: UIViewControllerAnimatedTransitioning {
         case .In:
           toView.alpha = 1
         case .Out:
-          fromView.alpha = 0
+          tempfromView?.alpha = 0
```

Comment:
That was an example if we want to use the same animators (in the final implementation, I don't the same), but in this case: when we present a controller, we don't always have a `fromView`, which will make it fails on the guard with the previous implementation, even for a `Fade` or `FadeIn`.

If we choose to keep going with the last implementation, I will rollback that code.